Presentation.Web/app/components/global-admin/global-admin-misc.controller.ts

```diff
@@ -135,12 +145,65 @@
                 } else {
                     notify.addInfoMessage("KLE opdatering stoppet!");
                 }
+            };
 
-            }
+            $scope.$watch("selectedUser", function (newVal, oldVal) {
+                if (newVal === oldVal || !newVal) return;
+
+                if (typeof newVal === "object")
+                    return;
+                $scope.userOrganizations = [];
+                userService.getUserOrganizations(newVal).then(res => {
+                    $scope.userOrganizations.pushArray(res);
+                    $scope.userSelected = true;
+                });
+            });
+
+            $scope.removeUser = (id: number) => {
+                const name = $scope.selectedUser.text;
+                const email = $scope.selectedUser.email;
+
+                //$state.go("global-admin-misc-user-delete", { id: id});
+                const nameAndEmail = `${$scope.selectedUser.text}, ${$scope.selectedUser.email}`;
+                if (confirm(`Er du sikker på, at du vil slette ${nameAndEmail}`)) {
+                    notify.addInfoMessage(`Sletter ${nameAndEmail}`);
+                    userService.deleteUser(id)
+                        .then(() => {
+                                notify.addSuccessMessage(`Sletter ${nameAndEmail}`);
+                                $scope.userSelected = false;
+                                $scope.selectedUser = null;
+                            }
+                        ).catch(ex => {
+                            console.log(ex);
+                            notify.addErrorMessage(`Fejl ifm. sletning af brugeren Sletter ${nameAndEmail}`);
+                            $scope.reload();
+                        });
+                    $state.reload();
+                }
+            };
 
             function toggleKleButtonsClickAbility(updateAvailButton: boolean, updateButton: boolean) {
                 $scope.KleUpdateAvailableButtonInteraction = updateAvailButton;
                 $scope.KleApplyUpdateButtonInteraction = updateButton;
             }
+
+            function getAvailableUsers() {
```

Comment:
rename to `getAvailableUserOptions()` (otherwise I would expect this method to return "users".. right`?